ada_feeding/ada_feeding/trees/move_to_dummy_tree.py

```diff
@@ -35,15 +34,10 @@ def __init__(
 
         Parameters
         ----------
-        action_type_class_str: The type of action that this tree is implementing,
-            e.g., "ada_feeding_msgs.action.MoveTo". The input of this action
-            type can be anything, but the Feedback and Result must at a minimum
-            include the fields of ada_feeding_msgs.action.MoveTo
         dummy_plan_time: How many seconds this dummy node should spend in planning.
         dummy_motion_time: How many seconds this dummy node should spend in motion.
         """
-        # Import the action type
-        self.action_type_class = import_from_string(action_type_class_str)
+        self.action_type_class = None
```

Comment:
Is this line necessary? It's a departure from the logic used in other MoveTo trees.

Reply:
Note (if you did it for the linter): idk I'm okay ignoring the linter warning about initializing an attribute outside of the the __init__ function. Where it makes sense I think we should, but there are other cases where it seems unnecessary, like here.

Reply:
I thought I did it for the linter, but it's not complaining. Removed.